README.md

```diff
@@ -180,3 +180,6 @@ then sort the list, which can be done by running the bash command:
 Note that only the FreeRTOS-Kernel Source Files, [include](include),
 [portable/MemMang](portable/MemMang), and [portable/Common](portable/Common)
 files are checked for proper spelling, and formatting at this time.
+
+## Third Party Tools
+[Link](.github/3P_TOOLS.md)
```

Comment:
```suggestion
FreeRTOS leverages third-party tools for tracing and static code analysis. Detailed information and a getting started guide are available at [third party tools](.github/3P_TOOLS.md).
```

Suggest we make it a sentence instead of just a link.